framework_tvb/tvb/core/services/algorithm_service.py

```diff
@@ -220,7 +220,10 @@ def get_upload_algorithms():
         """
         categories = dao.get_uploader_categories()
         categories_ids = [categ.id for categ in categories]
-        return dao.get_adapters_from_categories(categories_ids)
+        algorithm_list = dao.get_adapters_from_categories(categories_ids)
+
+        # We don't want TumorDatasetImporter to be shown among the other importers
+        return list(filter(lambda x: x.classname != 'TumorDatasetImporter', algorithm_list))
```

Comment:
This is not nice, to hard-code a hide with a specific name.
Why isn't this TumorData Algorithm a creator ? Then it won't appear in the web GUI, and can still be found in DB as an Algorithm to be launched from ProjectController.

Reply:
We always discussed about as an importer because it doesn't create anything new, but if it is okay to make it a creator, I will do that.

Reply:
indeed, from certain definition/language perspectives, this is an "importer", but when  that is becoming a problem in the code, it is ok to review the category.
to support the "creators" group, one can say this algorithm is "creating tumor tvb datatypes, from online bids data"